types/block.go

```diff
@@ -307,13 +307,18 @@ func MaxDataBytesNoEvidence(maxBytes int64, valsCount int) int64 {
 // computed from itself.
 // It populates the same set of fields validated by ValidateBasic.
 func MakeBlock(height int64, txs []Tx, lastCommit *Commit, evidence []Evidence) *Block {
+	var txKeys []TxKey
```

Comment:
nit: can make use of knowing len of txs again: `txKeys := make([]TxKey, 0, len(txs))`

Reply:
done!